storage/innobase/handler/handler0alter.cc

```diff
@@ -2815,6 +2815,15 @@ ha_innobase::check_if_supported_inplace_alter(
 		}
 	}
 
+	if (m_prebuilt->table->is_stats_table()) {
+		if (ha_alter_info->online) {
+			ha_alter_info->unsupported_reason =
+				"Online alter is not supported for "
+				"statistics table";
```

Comment:
For non-English translations of `ER_ALTER_OPERATION_NOT_SUPPORTED_REASON` it would be better to avoid any hard-coded English text. Let us check what the rest of the English error message is saying:
```
ERROR 0A000: LOCK=NONE is not supported. Reason: _. Try LOCK=SHARED
```
In the above reason string, the only non-redundant part is "statistics table". I think that we should use the table name (`mysql.innodb_table_stats` or `mysql.innodb_index_stats`) as the reason. In that way, the error message works also in other languages.